std_msgs/Header header
int32 marker_quantity
Marker[] markers
int32 rigid_body_quantity
RigidBody[] rigid_bodies